pages/linux/lftp.md

```diff
@@ -0,0 +1,31 @@
+# lftp
+
+> Sophisticated file transfer program.
+
+- Connect to an FTP server:
+
+`lftp {{ftp.example.com}}`
+
+- Download multiple files (glob expression):
+
+`mget {{*.png}}`
```

Comment:
```suggestion
`mget {{path/to/*.png}}`
```
Here we always put `path/to/` in front of files to indicate it can be a relative path. To keep consistent with other pages I suggest to add this.